docs/docs/tutorials/ssh-signing.md

```diff
@@ -0,0 +1,103 @@
+# :material-key: SSH and Commit Signing
+
+This tutorial will guide you through setting up SSH keys and commit signing for GitHub. SSH keys are a more secure way to connect to GitHub, rather than other protocols like HTTPS. Commit signing is used to verify that commits are coming from you and have not been tampered with.
+
+!!! warning
+    You must have a verified email address on GitHub to add SSH keys and sign commits.
+
+## Create an SSH Key
+
+=== "Windows"
+
+    In an admin PowerShell terminal, enter the following command:
+
+    ```text
+    ssh-keygen -t ed25519 -C "your_github_email@example.com"
+    ```
+
+    ??? Failure "Windows: `ssh-keygen` not recognized"
+
+        Older versions of Windows do not include SSH so it must be installed separately.
+
+        1. Open the __Settings__ app. Go to __System__ then __Optional Features__.
+        2. Click __Add a feature__ and search for "OpenSSH."
+        3. Install the feature. Restart your computer.
+        4. Add the OpenSSH directory to your PATH.
+
+            The default location is `C:\Windows\System32\OpenSSH`
+
+        Open a new terminal. You should be able to execute `ssh-keygen` and other `ssh` commands.
+
+    Save to the default location `c:/Users/YOU/.ssh/id_ed25519` and do not enter a passphrase. This will allow you to use the key without entering a password every time you push to GitHub.
+
+    Continue with the following commands:
+
+    ```text
+    type c:/Users/YOU/.ssh/id_ed25519 | clip
+    Get-Service -Name ssh-agent | Set-Service -StartupType Manual
+    Start-Service ssh-agent
+    ssh-add c:/Users/YOU/.ssh/id_ed25519
+    ```
+
+=== "Linux/Mac"
+
+    In a terminal window, enter the following command:
+
+    ```text
+    ssh-keygen -t ed25519 -C "your_github_email@example.com"
+    ```
+
+    Save to the default location `~/.ssh/id_ed25519` and do not enter a passphrase. This will allow you to use the key without entering a password every time you push to GitHub.
+
+    Continue with the following commands:
+
+    ```text
+    pbcopy < ~/.ssh/id_ed25519.pub
+    eval "$(ssh-agent -s)"
+    ssh-add ~/.ssh/id_ed25519
+    ```
+
+To add the key to your GitHub account, go to <https://github.com/settings/ssh/new>.
+
+- Name your key something descriptive.
+- Select "Authentication Key" as the key type.
```

Comment:
My commits were labelled `unverified` until I added my key as a "signing" key. What is the auth key for? Should we do both?

Reply:
TLDR: Authentication and signing keys are completely independent of each other. We can use SSH for singing only since authentication is not required. If you think we should add both and have people clone the repo on SSH instead of HTTPS, we can keep that section. Let me know.

---

Authentication is for interacting with GitHub over SSH instead of HTTPS. Many devs/companies prefer this but I would assume most of our members are only familiar with HTTPS.

i.e. The repo can be cloned by both:
- `https://github.com/macformula/racecar.git`
- `git@github.com:macformula/racecar.git`

and this changes the remote origin URL (`git config --get remote.origin.url`) you interact with, note they are still the same repo.

Signing keys just attach a stamp to your commits, which is why yours were showing `unverified` until you added one. Branch protection rules can be set up to force signed commits if that's a direction we'd like to take.

The reason many people's commits don't show unverified is because they have `Vigilant mode` disabled. Details [here](https://github.com/settings/keys) (scroll down).

Since we can use the same key for authentication and signing I thought it would be easy to set up. The problem might be converting people from the HTTPS to SSH. We could just have them set `git config --get remote.origin.url git@github.com:macformula/racecar.git`. This will work out of the box assuming they have generated and added an authentication SSH key to their GitHub.

Reply:
Does ssh authentication make us anymore secure? If our repo is public, what does it prevent? I see the purpose of signed commits, but not ssh authentication for our repo. 

I don't want to make it more difficult for anyone to clone our repo so I want html cloning to be an option. Does it make sense to force all of our developers to use ssh? For what purpose?

Reply:
You could add a "converting to ssh" section to the bottom of your page with this command

Reply:
TLDR: The best solution is to add the key as both authentication and signing. Converting from HTTPS to SSH is optional.

---

Since the repo is public, using SSH is not essential. We don't have force it but I would still recommend the key be added to GitHub accounts for both authentication and signing because:

- `ssh -T git@github.com` will not work unless GitHub has an authentication key

- Commit signing works without an authentication key but requires a GitHub and/or PC passwords each time you try to push commits

Reply:
Added the converting to ssh (Set Repo to Use SSH) section at the end